packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -11,6 +11,8 @@ import {
   SpinLoader,
   SignalPopover,
   PerformanceSignals,
+  MenuItem,
+  SplitButton,
```

Comment:
Within [designs](https://www.figma.com/file/pPrSfGhbWJvA6G6NiAU4Y2/EXPO-2381%3A-Compass-Search-Indexes?type=design&node-id=936-128912&mode=design&t=wPgyG1Zn60iBTz2M-0), its not a split button, but a regular dropdown button

Reply:
You are right, I didn't see the difference until you told me. Fixing it.